test/postgresql.initialization.test.js

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2015. All Rights Reserved.
+// Node module: loopback-connector-postgresql
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var should = require('should');
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var postgresql = require('../lib/postgresql.js');
+
+describe('initialization', function() {
+  it('should take pg-pool setting', function(done) {
+    // settings can be empty because we do not actually need to connect
+    var settings = {};
+    var dataSource = new DataSource(require('../'), settings);
+    dataSource.settings.max = 999; // non-default value
```

Comment:
Verifying the pool state before running the test will require initializing with defaults. I chose 999 as a non-default value that is very unlikely to ever become the default––currently 10.